lib/preload.js

```diff
@@ -66,10 +108,397 @@ const bindMount = (source, target) => {
   }
 }
 
-const createContainer = (docker, options) => {
-  const mounts = [bindMount(options.image, DISK_IMAGE_PATH_IN_DOCKER)]
-  if (options.splashImage) {
-    mounts.push(bindMount(options.splashImage, SPLASH_IMAGE_PATH_IN_DOCKER))
+const zipContainsFiles = (archive, files) => {
+  // FIXME: read from the zip directory instead of readin the whole file
+  // This is broken in unzipper for now: an invalid zip file will throw an
+  // uncaught error event.
```

Comment:
Is this something that we can use and then catch the error?

Reply:
It will emit an error here https://github.com/ZJONSSON/node-unzipper/blob/master/lib/PullStream.js#L66 .
We don't have access to this stream with `node-unzipper`'s api so we can't handle it

Reply:
I mean if we use
```javascript
return unzipper.Open.file(archive)
  .then(function(d) {
    ...
  })
```

Reply:
and `node-unzipper` soesn't handle it either.